flink-examples/src/main/scala/io/pravega/examples/flink/iot/scala/TurbineHeatProcessor.scala

```diff
@@ -71,22 +62,22 @@ object TurbineHeatProcessor {
   }
 
   def main(args: Array[String]) {
-    val params = Parameters(ParameterTool.fromArgs(args))
+
+    val params = ParameterTool.fromArgs(args)
+    val helper = new FlinkPravegaParams(params)
 
     // ensure that the scope and stream exist
-    ensureStream(params)
+    val stream = helper.createStreamFromParam("stream", "examples/turbineHeatTest")
```

Comment:
Since we're changing the semantic of the parameter anyway, how about renaming the parameter to `input` rather than `stream`?   The output is named `output` so there's nice symmetry.